device-server/src/main/kotlin/com/badoo/automation/deviceserver/data/XcuiTestExecutionConfig.kt

```diff
@@ -0,0 +1,21 @@
+package com.badoo.automation.deviceserver.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import java.nio.file.Path
+
+data class XcuiTestExecutionConfig(
+        @JsonProperty("app_name")
+        val appName: String,
+
+        @JsonProperty("xctestrun_file_name")
+        val xctestrunFileName: String,
+
+        @JsonProperty("test_name")
+        val testName: String,
+
+        @JsonProperty("path_to_dir_with_xctestrun_file")
+        val pathToDirWithXctestrunFile: Path,
```

Comment:
Can we merge this property with `xctestrunFileName`, it does not look like we are using them separately anywhere.

Reply:
Done.
#36